scripts/umbrel-os/external-storage/monitor

```diff
@@ -14,6 +14,12 @@ check_if_not_already_running() {
   fi
 }
 
+get_uas_string () {
+  ID_VENDOR=$(udevadm test /block/${block_device} | grep "ID_VENDOR_ID" | sed 's/ID_VENDOR_ID=//')
```

Comment:
If I run this on either of my Pis:

```
sudo udevadm test /block/sda | grep "ID_VENDOR_ID"
```

I just get loads of stuff printed to stderr and nothing matching `ID_VENDOR_ID`. Is that expected?

Reply:
> I just get loads of stuff printed to stderr

Yeah, that's normal, udevadm test is quite verbose.

> and nothing matching ID_VENDOR_ID. 

That isn't normal.

Are you testing this on a Pi with an SSD connected? Because I think it might not work on a normal PC with an internal SSD.

```
ID_VENDOR=$(udevadm test /block/sda | grep "ID_VENDOR_ID" | sed 's/ID_VENDOR_ID=//')
ID_MODEL=$(udevadm test /block/sda | grep "ID_MODEL_ID" | sed 's/ID_MODEL_ID=//')
UAS_RULE="${ID_VENDOR}:${ID_MODEL}:u"
echo $UAS_RULE
```

Reply:
Sorry, I didn't read the first sentence

Reply:
Oh, I was running as root which fails, because this script will be running as root.

I've just tested and running as non-root does indeed work.

Doesn't that mean this script will always fail at boot?

Reply:
```
umbrel@umbrel:~/umbrel $ udevadm test /block/sda 2>/dev/null | grep "ID_VENDOR_ID"
ID_VENDOR_ID=04e8
umbrel@umbrel:~/umbrel $ sudo udevadm test /block/sda 2>/dev/null | grep "ID_VENDOR_ID"
umbrel@umbrel:~/umbrel $
```